js_modules/dagster-ui/packages/ui-core/src/instance/useBulkActionStatusFilter.tsx

```diff
@@ -0,0 +1,50 @@
+import {BulkActionStatus} from '../graphql/types';
+import {useStaticSetFilter} from '../ui/BaseFilters/useStaticSetFilter';
+
+const labelForBackfillStatus = (key: BulkActionStatus) => {
+  switch (key) {
+    case BulkActionStatus.CANCELED:
+      return 'Canceled';
+    case BulkActionStatus.CANCELING:
+      return 'Canceling';
+    case BulkActionStatus.COMPLETED:
+      return 'Completed';
+    case BulkActionStatus.FAILED:
+      return 'Failed';
+    case BulkActionStatus.REQUESTED:
+      return 'In progress';
+    case BulkActionStatus.COMPLETED_SUCCESS:
+      return 'Success';
+    case BulkActionStatus.COMPLETED_FAILED:
+      return 'Failed';
+  }
+};
```

Comment:
Could use a `Record<BulkActionStatus, string>` instead to get type exhaustiveness.

Reply:
Hmm good catch this is definitely awkward, will switch it.